JourneysSquad/JourneysSquad/View/AddBillView/DataInputFieldView.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+/// A field to enter text with a label above one.
+struct DataInputFieldView: View {
+    /// The text entered in the field.
+    @Binding var input: String
+
+    /// The label displayed above the field.
+    let labelOfField: String
+
+    /// The placeholder text displayed within the field.
+    let placeholder: String
+
+    /// The type of keyboard to display when the field is tapped.
+    let keyboardType: UIKeyboardType
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(labelOfField.uppercased())
+                .font(.system(size: fontBodySize, weight: .bold, design: .rounded))
```

Comment:
```suggestion
                .font(.system(size: fontBodySize, weight: .bold, design: .default))
```

Our [App style guidelines](https://github.com/ios-course/link-team-project/wiki/App-style-guidelines) doesn't contain font `.rounded`
Please, review the rest of code.

Reply:
Our guide contains the type of font. It's actually `default`. Our guide doesn't contain info about `design` and `weight`. 
May be we need to correct our fonts? I have some suggestions, but in this case I don't see violation of out style guide. 
What do you think about it?

Reply:
Yep, my bad. 
You can discuss your suggestions with team and after that add it to the [App style guidelines](https://github.com/ios-course/link-team-project/wiki/App-style-guidelines).

Reply:
Fixed